pkg/apis/toolchain/v1alpha1/masteruserrecord_types.go

```diff
@@ -86,13 +91,25 @@ type UserAccountStatusEmbedded struct {
 	UserAccountStatus `json:",inline"`
 }
 
+type Cluster struct {
+	// Name is the name of the corresponding KubeFedCluster resource
+	Name string `json:"name"`
+
+	// APIEndpoint is the API Endpoint of the cluster
+	APIEndpoint string `json:"apiEndpoint"`
+
+	// ConsoleURL is the web console URL of the cluster
+	ConsoleURL string `json:"consoleURL"`
+}
+
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
 // MasterUserRecord is the Schema for the masteruserrecords API
 // +k8s:openapi-gen=true
 // +kubebuilder:subresource:status
 // +kubebuilder:resource:scope=Namespaced
 // +kubebuilder:resource:shortName=mur
+// +kubebuilder:printcolumn:name="Cluster",type="string",JSONPath=`.spec.userAccounts[].targetCluster`
```

Comment:
shouldn't that user rather the `Cluster` param/object instead of the `targetCluster` string when it's deprecated?
```suggestion
// +kubebuilder:printcolumn:name="Cluster",type="string",JSONPath=`.spec.userAccounts[].cluster.name`
```

Reply:
This PR doesn't change the Target Cluster (string) from UserAccount *Spec*: `.spec.userAccounts[].targetCluster`. It basically says that User Account should be provisioned on that particular cluster name.

What is changed here is the status of UserAccount. When it's provisioned it now has a detailed cluster object with URLs.

Does it make sense to you?

Reply:
Yes, you are right. Sorry for confusion